jbpm-task-assigning/jbpm-task-assigning-core/src/test/java/org/jbpm/task/assigning/model/solver/PriorityHelperTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.task.assigning.model.solver;
+
+import java.util.Random;
+import java.util.stream.Stream;
+
+import org.jbpm.task.assigning.model.solver.PriorityHelper;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class PriorityHelperTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void isHighLevelTest() {
+        Stream.of(0, 1, 2).forEach(level -> assertTrue(PriorityHelper.isHighLevel(level)));
+    }
+
+    @Test
+    public void isMediumLevelTest() {
+        Stream.of(3, 4, 5, 6).forEach(level -> assertTrue(PriorityHelper.isMediumLevel(level)));
+    }
+
+    @Test
+    public void isLowLevelTest() {
+        Stream.of(7, 8, 9, 10).forEach(level -> assertTrue(PriorityHelper.isLowLevel(level)));
+    }
+
+    @Test
+    public void calculateWeightedPenaltySuccessfulTest() {
+        int priority = new Random().nextInt(11);
```

Comment:
There is hardly ever a reason to use `Random` in tests as we want to have tests 100% stable.

Reply:
This random test were useful to me during development, and I don't believe they might cause issues. But what I can to is to keep them here but  @Ignored so I can go back to them when I need or want to run them locally.

Reply:
Ah, sorry, this particular random test can be made non random. I was refering to some other tests that do random operations on the solver.